### ISSUE TITLE
Missing type for requestOptions property in BackendOptions

### ISSUE THREAD

**Issue by emsa16** [i18next/i18next-http-backend#67]
## 🐛 Bug Report

The `requestOptions` property in `BackendOptions` cannot be used in ts files due to its type missing in `index.d.ts`

## To Reproduce

```ts
    type RequestType = BackendOptions['request'];
    const getTranslations: RequestType = (
      options,
      url,
      payload,
      callback,
    ) => {
      const {requestOptions} = options; // TypeScript error: Property 'requestOptions' does not exist on type 'BackendOptions'.
    };

```

## Expected behavior

No error

## Your Environment

- *runtime version*: VS Code 1.56.1, TypeScript 3.8.3
- *i18next-http-backend version*: 1.2.4
- *os*: Mac

**Comment by adrai** [comment]
Would you like to send a Pull Request to address this?

**Comment by emsa16** [comment]
I can try :)

**Comment by adrai** [comment]
@pedrodurek can you review then?

**Comment by adrai** [comment]
fixed with v1.2.5

**Comment by emsa16** [comment]
Thanks for fixing it and sorry for dropping the ball, to busy at work
